API.md

```diff
@@ -355,43 +355,46 @@ public struct HTTPResponseStatus: Equatable, CustomStringConvertible, Expressibl
 }
 
 /// HTTP Methods handled by http_parser.[ch] supports
-public enum HTTPMethod : RawRepresentable {
-    /* be future-proof, http_parser can be upgraded */
-    case other(String)
+public struct HTTPMethod : RawRepresentable {
```

Comment:
Yeah, I was wondering if having `RawRepresentable` really added any value here, particularly as having the `rawValue` label in the `init` is really non-intuitive/non-Swifty (hence taking the approach in `Notification.Name` of having two constructors - with and without `rawValue`)